src/translator_en.h

```diff
@@ -394,6 +394,10 @@ class TranslatorEnglish : public Translator
       {
         return "Data Structure Documentation";
       }
+      else if (Config_getBool(OPTIMIZE_OUTPUT_VHDL))
+      {
+          return "Design Unit Documentation";
+      }
```

Comment:
As is this should be OK, though for update consistency with other output languages I have a bit of a doubt whether or not it wouldn't be better to call here a function `trDesignUnitDocumentation` do this can be in a category "new since 1.8.19" with possibilities to overload nicely (compare with e.g. `trDesignUnitHierarchy`).
@doxygen your idea ?

Reply:
As for the German translation there already is a VHDL switch in trClassDocumentation I went that way also for English. The problem with the approach with adding an extra function trDesignUnitDocumentation is that trClassDocumentation is used in many places which all would have to be adapted when using a separate function.

Reply:
It is indeed consistent with the German translation and I see that in the Dutch (and other) translation the part `if (Config_getBool(OPTIMIZE_OUTPUT_FOR_C))` is missing, all in all not nice.

The intention I had was  to have the code as:
```
    virtual QCString trCompoundIndex()
    {
      if (Config_getBool(OPTIMIZE_OUTPUT_FOR_C))
      {
        return "Data Structure Index";
      }
      else if (Config_getBool(OPTIMIZE_OUTPUT_FOR_VHDL))
      {
        return trDesignUnitDocumentation();
      }
      else
      {
        return "Class Index";
      }
    }

```
and have this "else if" part in all translators so that as soon as a translator implements `trDesignUnitDocumentation` it is used up (still a number of changes to be done now though).

Reply:
Ah, I see. I will do so and update my pull request